scripts/run.sh

```diff
@@ -22,6 +22,7 @@ function cleanup {
 # NATS
 ###
 gnatsd &
+sleep 1
```

Comment:
Are we sure that 1 seconds is enough so that NATS is up and running? There is a possibility that we end up in a "works on my machine" situation.
If we need to wait, wouldn't it be better if we query one of the NATS's [monitoring endpoits](https://nats-io.github.io/docs/nats_server/monitoring.html) `/varz` for example and check the response

Reply:
I've changed checking availability